### PR TITLE
Added test cases and input data for the parenthesis operator.

### PR DESCRIPTION
**Pull request by Garee**

### CODE REVIEW

File: spec_parens.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                {
+                    "id": "8",
+                    "freezeDate" : "2015-07-20T12:00:00.000Z",
+                    "currentUser" : null,
+                    "context": "When asked for all tasks with priority 1 that are overdue or due today in input_parens.json.",
+                    "query": "p1 & (overdue | today)",
+                    "testFile": "input_parens.json",
+                    "assertSections": [
+                        {
+                            "name": "Priority 1 & (Overdue | Today)",
```
Comment (adsantos) [inline spec_parens.json]: Can you add a test case for `(p1 & overdue) | today` and for `p1 & overdue | today`, making sure that the test data will result in a different output than when querying `p1 & (overdue | today)`?

File: spec_parens.json [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                {
+                    "id": "5",
+                    "freezeDate" : null,
+                    "currentUser" : null,
+                    "context": "When asked for all tasks that do not have priority 1 in input_parens.json.",
+                    "query": "(!p1)",
+                    "testFile": "input_parens.json",
+                    "assertSections": [
+                        {
+                            "name": "!Priority 1",
```
Comment (adsantos) [inline spec_parens.json]: I suppose the `!` will be represented in a different way, in the section name, but don't change it, let's wait for the copy team.

File: spec_parens.json (hunk 1 above)
Comment (Garee) [inline spec_parens.json]: Done.

Comment (adsantos) [inline spec_parens.json]: Could you also add `p1 & overdue | today`, just to show that it returns the same as `(p1 & overdue) | today`?

Comment (Garee) [inline spec_parens.json]: Done, sorry I missed that in the previous comment.

**Pull request merged**